### PR TITLE
network: add `neogo_version` metric, deprecate `serv_node_version`

### LINKED ISSUE #2999: improve version metric
improve version metric please, like this one, for example:
```
vm_app_version{version="vmagent-20230407-010133-tags-v1.90.0-0-gb5d18c0d2", short_version="v1.90.0"} 1
```
not this:
```
neogo_serv_node_version{description="Node version: ",value="/NEO-GO:0.101.0/"} 0
```

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Close #2999.

### CODE REVIEW

File: pkg/network/prometheus.go [hunk 1]
```diff
@@ -34,6 +35,14 @@ var (
 		[]string{"description", "value"},
 	)
 
+	neogoVersion = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Help:      "NeoGo version and network server ID",
+			Name:      "neogo_version",
+			Namespace: "neogo",
+		},
+		[]string{"version", "server_id"})
```
Comment (roman-khimov) [inline pkg/network/prometheus.go:44]: Do we want to mix `server_id` in? It's a separate thing to me.

Comment (AnnaShaleva) [reply pkg/network/prometheus.go]: Fixed.

File: pkg/network/prometheus.go [hunk 2]
```diff
@@ -34,6 +35,22 @@ var (
 		[]string{"description", "value"},
 	)
 
+	neogoVersion = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Help:      "NeoGo version",
+			Name:      "neogo_version",
```
Comment (roman-khimov) [inline pkg/network/prometheus.go:41]: `neogo_neogo_version`?

Comment (AnnaShaleva) [reply pkg/network/prometheus.go]: Well, it's just because we have `neo_go` namespace. But you're right, it's too much, fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**